features/eidas_user_journeys.feature

```diff
@@ -8,7 +8,7 @@ Feature: eIDAS user journeys
     And they start an eIDAS journey
     And they select eIDAS scheme "Stub IDP Demo"
     Then they should be at IDP "Stub Country"
-    Given they go back to the country picker
+    Given they choose to go back to the "choose-a-country" page
```

Comment:
I don't think this is a good idea - the point of cucumber tests is that they should be readable by non-tech people, such as PMs and DMs. It's easier for them to refer to the "country picker" rather than "choose-a-country" page - the URL is arbitrary. At the very least, can we have a new method that would map the symbolic page descriptions to specific URLs, something like `they choose to go to the {page_name} page` - and in that method join the URL like we do now but map the name to the specific page URL instead of inserting the URL directly.

Reply:
Point noted and changes made accordingly. Can you please review again. Thanks